packages/atproto_client/models/string_formats.py

```diff
@@ -52,19 +63,49 @@
 
 
 def only_validate_if_strict(validate_fn: Callable[..., str]) -> Callable[..., str]:
-    """Skip pydantic validation if not opting into strict validation via context."""
+    """Skip pydantic validation if not opting into strict validation via context.
+
+    :param validate_fn: The validation function to conditionally apply
+    :type validate_fn: Callable[..., str]
+    :return: A wrapped validation function that only validates in strict mode
+    :rtype: Callable[..., str]
```

Comment:
the entire project is documented in Google Docstring Style; could you pls align it? here is the guide: https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings (or just see how other paths of the SDK is documented)